src/ModuleList/Lesson/Lesson.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
 
+import { useLesson } from './useLesson';
+
 interface Params {
   moduleId: string;
   lessonId: string;
 }
 
 const Lesson = () => {
   const { moduleId, lessonId } = useParams<Params>();
+  const lessonState = useLesson(lessonId);
```

Comment:
I'm not entirely sure we need the `useLesson` hook here... at least not at the moment. We could just use the `useLessonMeta` hook that we created before. We really only would need a custom hook when we need to handle more complex logic. What are your thoughts?

```suggestion
  const { data, isLoading } = useLessonMeta(Number(lessonId));
```

If you make this change, see comment on line `23` for what you would change that line to.

Reply:
![image](https://user-images.githubusercontent.com/4872808/97759996-9598a880-1ad8-11eb-8dce-a85ff93bc880.png)
Is this acceptable?

Reply:
I just removed it since it broke build, we can add it back later if need be

Reply:
Ah yeah removing it was the right choice. I more so added it to show that you can still use the `isLoading` variable from the `useLessonMeta` 😅